src/coreclr/jit/compiler.h

```diff
@@ -6723,6 +6773,8 @@ class Compiler
     PhaseStatus optFindLoopsPhase(); // Finds loops and records them in the loop table
 
     void optFindLoops();
+    void optFindNewLoops();
```

Comment:
This name is unfortunately confusing: when not in context, I thought it meant "find newly created loops, after some initial loop finding pass".

Maybe this is temporary and we'll get rid of "old" at some point and rename "new" to remove the word "new" everywhere.

Reply:
Right, we should be able to get rid of these old/new distinctions soon.